### ISSUE TITLE
First member of team not recognised

### ISSUE THREAD

**Issue by nickynicolson** [bionomia/dwc_agent#12]
Using: command line utility to access the `dwcagent` gem

Version: `v3.0.0.6`

Input: `da Torre A.R. & Correia M.P`

Expected result:
```{json}
[{"family":"Torre","given":"A.R.","suffix":null,"particle":"da","dropping_particle":null,"nick":null,"appellation":null,"title":null}
,{"family":"Correia","given":"M.P.","suffix":null,"particle":null,"dropping_particle":null,"nick":null,"appellation":null,"title":null}]
```

Actual result:
```{json}
[{"family":"Correia","given":"M.P.","suffix":null,"particle":null,"dropping_particle":null,"nick":null,"appellation":null,"title":null}]
```

**Comment by dshorthouse** [comment]
Apologies for the delay. For some reason, I did not have notifications enabled in my own repo! It's the cleaning routine that runs post-parse that's causing this; it's confused by the lack of punctuation.

Input: `da Torre, A.R. & Correia, M.P.`

...works as expected. The absence of commas in reverse ordering of names is a bugger.

**Comment by dshorthouse** [comment]
Fixed in v3.0.0.8
